### PR TITLE
ledger-tool: Run rustfmt with format_strings = true

### PR DESCRIPTION
**Pull request by steviez**
This PR spun out from https://github.com/solana-labs/solana/pull/34274 per some conversation to consider running the `rustfmt` on a subset to see if it makes things more manageable.

#### Problem
Long string literals can cause rustfmt to fail, which results in rustfmt failing to format entire functions. There are several instances of this in ledger-tool, so format these files with wrapped strings so that formatting will apply to functions again.

#### Summary of Changes
Note that this PR was created by adding format_strings = true to rustfmt.toml; however, this change does NOT persist that rule as the rule would format the entire repo.

While reviewing the diff, I do see some lines that were wrapped that could be candidates for inlining the values in format strings:
https://github.com/solana-labs/solana/blob/4832b4ecec57f9e87a640011b30bc5cc07921721/ledger-tool/src/bigtable.rs#L464

However, there are similar instances in lines that are not wrapped, so I'm inclined to defer that

### CODE REVIEW

File: ledger-tool/src/main.rs [hunk 1]
```diff
@@ -1398,702 +1422,756 @@ fn main() {
...
+                        ),
+                )
+                .arg(
+                    Arg::with_name("output_directory")
+                        .index(2)
+                        .value_name("DIR")
+                        .takes_value(true)
+                        .help(
+                            "Output directory for the snapshot [default: --snapshot-archive-path \
+                             if present else --ledger directory]",
```
Comment (CriesofCarrots) [inline ledger-tool/src/main.rs:1810]:
```suggestion
                            "Output directory for the snapshot \
                            [default: --snapshot-archive-path if present else --ledger directory]",
```
If you're keen to match what I did in CLI, and this is short enough to not break fmt.

File: ledger-tool/src/main.rs [hunk 2]
```diff
@@ -1398,702 +1422,756 @@ fn main() {
...
+                .arg(
+                    Arg::with_name("bootstrap_stake_authorized_pubkey")
+                        .long("bootstrap-stake-authorized-pubkey")
+                        .value_name("BOOTSTRAP STAKE AUTHORIZED PUBKEY")
+                        .takes_value(true)
+                        .validator(is_pubkey_or_keypair)
+                        .help(
+                            "Path to file containing the pubkey authorized to manage the \
+                             bootstrap validator's stake [default: --bootstrap-validator \
+                             IDENTITY_PUBKEY]",
```
Comment (CriesofCarrots) [inline ledger-tool/src/main.rs:1864]:
```suggestion
                             bootstrap validator's stake \
                             [default: --bootstrap-validator IDENTITY_PUBKEY]",
```

File: ledger-tool/src/main.rs [hunk 3]
```diff
@@ -1398,702 +1422,756 @@ fn main() {
...
+                    Arg::with_name("start_slot")
+                        .index(1)
+                        .value_name("SLOT")
+                        .takes_value(true)
+                        .required(true)
+                        .help("Start slot to purge from (inclusive)"),
+                )
+                .arg(Arg::with_name("end_slot").index(2).value_name("SLOT").help(
+                    "Ending slot to stop purging (inclusive) [default: the highest slot in the \
+                     ledger]",
```
Comment (CriesofCarrots) [inline ledger-tool/src/main.rs:2087]:
```suggestion
                    "Ending slot to stop purging (inclusive) \
                    [default: the highest slot in the ledger]",
```

**Review by CriesofCarrots — APPROVED** [review]
I suggested a couple `default` reformats to match what I did in solana-cli, but I don't feel strongly about them here. Feel free to merge as-is if you prefer.

**Comment by steviez** [comment]
> I suggested a couple `default` reformats to match what I did in solana-cli, but I don't feel strongly about them here. Feel free to merge as-is if you prefer.

No, these are good ones that I missed on my pass through (think my eyes were starting to bleed from doing `solana-cli` and this one back-to-back haha).

**Review by CriesofCarrots — APPROVED** [review]

**Pull request merged**